opensrp-tasking/src/main/java/org/smartregister/tasking/interactor/BaseInteractor.java

```diff
@@ -398,15 +398,7 @@ public void saveCaseConfirmation(JSONObject jsonForm, String eventType) {
         TaskingLibrary.getInstance().getTaskingLibraryConfiguration().saveCaseConfirmation(this, presenterCallBack, jsonForm, eventType);
     }
 
-    protected String getMemberTasksSelect(String mainCondition, String[] memberColumns) {
```

Comment:
Is this code moved to the appropriate implementation on Reveal? The idea is to import this library to Reveal

Reply:
We need an issue for moving this method